docs/src/services/proxy/metrics.md

```diff
@@ -47,29 +47,33 @@ The proxy exposes the following general metrics:
   The number of currently active upstream endpoints. Note that this tracks the number of endpoints that the proxy
   knows of rather than those that it is connected to (see [Session Metrics][session-metrics] instead for those)
 
-* `quilkin_bytes_total{event}`
+* `quilkin_bytes_total{event, asn, ip_prefix}`
 
    The total number of bytes sent or recieved
   * The `event` label is either:
     * `read`: when the proxy receives data from a downstream connection on the listening port.
     * `write`: when the proxy sends data to a downstream connection via the listening port.
 
-* `quilkin_packets_total{event}`
+* `quilkin_packets_total{event, asn, ip_prefix}`
 
   The total number of packets sent or recieved.
   * The `event` label is either:
     * `read`: when the proxy receives data from a downstream connection on the listening port.
     * `write`: when the proxy sends data to a downstream connection via the listening port.
 
-* `quilkin_errors_total{event}`
+* `quilkin_packet_jitter{event, asn, ip_prefix}`
+
+  The time between receiving new packets.
+
+* `quilkin_errors_total{event, asn, ip_prefix}`
 
   The total number of errors encountered while reading a packet from the upstream endpoint.
 
 ## Session Metrics
 
 The proxy exposes the following metrics around sessions:
 
-* `quilkin_session_active{asn}{ip_prefix}`
+* `quilkin_session_active{asn, organization, cc, ip_prefix, prefix_entity, prefix_name}`
```

Comment:
What's "cc" here - it's the only items I couldn't find some docs on above.

Reply:
Ah, it's "country code" I'll update the field to be more clear.